product_supplierinfo_discount/README.rst

```diff
@@ -47,6 +47,7 @@ Contributors
 ------------
```

Comment:
"This module allows to set up a discount in the supplier info form, and propagate
 it to the purchase orders, with two advantages over pricelists:
 
 * The discount is directly added in the purchase orders instead of modifying the
   unit price.
 * Prices and discounts can be set on the same screen."